src/backend/distributed/shared_library_init.c

```diff
@@ -122,10 +122,6 @@ static bool StatisticsCollectionGucCheckHook(bool *newval, void **extra, GucSour
 static void CitusAuthHook(Port *port, int status);
 
 
-/* static variable to hold value of deprecated GUC variable */
```

Comment:
I think should have specific `DESCRIPTION` per removal for the users on the PR so that it can be picked up by the release process and gets into release notes